lib/utils/formatDate.ts

```diff
@@ -8,3 +8,18 @@ export const formatDate = (dateString: string): string => {
     return dateString; // Fallback to original string if parsing fails
   }
 };
+
+// Format date for scheduled actions with time (e.g. "Jan 15, 2:30 PM")
+export const formatScheduledActionDate = (dateString: string | null): string => {
+  if (!dateString) return "Never";
+  try {
+    return new Date(dateString).toLocaleDateString("en-US", {
+      month: "short",
+      day: "numeric",
+      hour: "2-digit",
+      minute: "2-digit",
+    });
+  } catch {
+    return "Invalid date";
+  }
+};
```

Comment:
Single Responsibility Principle
- actual: 2 functions exported from one file
- required: each file exports precisely `1 function`
- please create a new file for `formatScheduledActionDate`